### ISSUE TITLE
Terminology (JOSS Review)

### ISSUE THREAD

**Issue by fccoelho** [dylan-profiler/visions#24]
Hi, 
this is related to [JOSS review](https://github.com/openjournals/joss-reviews/issues/2145) of this project.

In the paper, you use the term "physical representation" of data. Although I can understand its meaning, in the context of the paper, I'd suggest using a more specific terminology, such as 'machine representation' because data frequently also has actual physical (as in Physics) representations which have nothing to do with the types a given hardware platform can deal with. 
For example: if you have a variable which stores temperature values, its "machine representation" is a Floating point number, but its physical units can be represented in celsius, fahrenheit, kelvin, etc. These units are closer to a [physical representation](https://en.wikipedia.org/wiki/Dimensional_analysis#A_more_complex_example:_energy_of_a_vibrating_wire), than the data type stored in the computer, and  the units  have huge semantic relevance to any analysis.

There are [libraries](https://pint.readthedocs.io/en/0.10.1/) which seek to represent the semantic of physical units to allow for correct conversion between values. 

In summary, I suggest to use a different terminology to refer to data types than "physical representation" because it can be misleading to certain users, particularly from the domain of scientific computing.

**Comment by sbrugman** [comment]
The paper and docs are updated, we'll rerender when all PRs are merged.
